Library/Homebrew/requirements/xcode_requirement.rb

```diff
@@ -21,13 +21,14 @@ def xcode_installed_version
   def message
     version = " #{@version}" if @version
     message = <<~EOS
-      A full installation of Xcode.app#{version} is required to compile
+      A full installation of Xcode.app#{version} is required for
       this software. Installing just the Command Line Tools is not sufficient.
     EOS
     unless xcode_swift_compatability?
       message += <<~EOS
 
-        Xcode >=10.2 requires macOS >=10.14.4 to build many formulae.
+        For Xcode >=10.2, we require macOS >=10.14.4
+        because it is necessary to build many formulae.
```

Comment:
Leave this text as-is because it's not clarifying and is slightly incorrect.

Reply:
The current text is false and unclear:

* According to https://developer.apple.com/documentation/xcode_release_notes/xcode_10_2_release_notes , _"Xcode 10.2 requires a Mac running macOS 10.14.3 or later."
* It implies that macOS >=10.14.4 is not a hard requirement but depends on the formula. But after https://github.com/Homebrew/brew/commit/da392cd9e836cf7955e9217d7e06b7a411dab107 and https://github.com/Homebrew/brew/commit/426c6e2c3f24ec48c33c3937c96ae76ca786a495 , Homebrew makes it a hard requirement. And this requirement comes from Homebrew rather than Apple.

Reply:
The current text is false and unclear:

* According to https://developer.apple.com/documentation/xcode_release_notes/xcode_10_2_release_notes , _"Xcode 10.2 requires a Mac running macOS 10.14.3 or later."_
* The text implies that macOS >=10.14.4 is not a hard requirement but depends on the formula. But after https://github.com/Homebrew/brew/commit/da392cd9e836cf7955e9217d7e06b7a411dab107 and https://github.com/Homebrew/brew/commit/426c6e2c3f24ec48c33c3937c96ae76ca786a495 , Homebrew makes it a hard requirement. And this limitation comes from Homebrew rather than Apple.

Reply:
It is not false. See https://github.com/Homebrew/brew/pull/5940 onwards.

Reply:
Sorry, https://github.com/Homebrew/brew/pull/5940#issuecomment-477583315 for the specific comment link.

Reply:
https://github.com/Homebrew/brew/pull/5940#issuecomment-477583315 only clarifies that Homebrew started to require 10.14.4 for formulae that `depend_on :xcode` to work around a bug in Xcode 10.2 that affects Swift apps. Well, I can reference that bug in the text.

The false statements in the current text are:

* _"[R]equires macOS >=10.14.4 to build many formulae"_ implies that macOS >=10.14.4 is not a hard requirement (for formulae that `depends_on :xcode` if installed XCode version is 10.2) but rather that some formula that `depends_on :xcode` will fail to build/function under these circumstances.
    * That is not so: the above-mentioned commits do make it a hard requirement which is enforced before ever trying to compile or install the formula, contradicting this text.
* _"Xcode >=10.2 requires macOS >=10.14.4"_ implies that this requirement comes from Xcode itself, i.e. Apple (e.g. maybe some XCode 10.2 features are disabled under 10.14.3).
    * While in fact, that is not so, and it's rather imposed by Homebrew privately (to work around a bug in Xcode 10.2).

I'll provide a new text.